autogpt_platform/backend/backend/blocks/jina/_auth.py

```diff
@@ -0,0 +1,39 @@
+from typing import Literal
+
+from autogpt_libs.supabase_integration_credentials_store.types import APIKeyCredentials
+from pydantic import SecretStr
+
+from backend.data.model import CredentialsField, CredentialsMetaInput
+
+JinaCredentials = APIKeyCredentials
+JinaCredentialsInput = CredentialsMetaInput[
+    Literal["jina"],
+    Literal["api_key"],
+]
+
+
+def JinaCredentialsField() -> JinaCredentialsInput:
+    """
+    Creates a Jina credentials input on a block.
+
+    """
+    return CredentialsField(
+        provider="jina",
+        supported_credential_types={"api_key"},  # noqa
```

Comment:
drop the noqa, shouldn't be needed

Reply:
it's not breaking but it does give me warnings without it

Reply:
<img width="899" alt="Screenshot 2024-10-23 at 7 29 19 PM" src="https://github.com/user-attachments/assets/0037c028-4d9e-4c1e-9188-4802019605b4">

Reply:
removed